lldb/source/Core/Module.cpp

```diff
@@ -1203,6 +1203,33 @@ bool Module::IsSwiftCxxInteropEnabled() {
   }
   return m_is_swift_cxx_interop_enabled == eLazyBoolYes;
 }
+
+bool Module::IsEmbeddedSwift() {
+  switch (m_is_embedded_swift) {
+  case eLazyBoolYes:
+    return true;
+  case eLazyBoolNo:
+    return false;
+  case eLazyBoolCalculate:
+    auto *sym_file = GetSymbolFile();
+    if (!sym_file)
+      return false;
+
+    m_is_embedded_swift = eLazyBoolNo;
+    auto options = sym_file->GetCompileOptions();
+    for (auto &[_, args] : options) {
+      for (const char *arg : args.GetArgumentArrayRef()) {
+        if (strcmp(arg, "-enable-embedded-swift") == 0) {
+          m_is_embedded_swift = eLazyBoolYes;
+          break;
```

Comment:
if this `break` is intended to break out of the outer loop, you will need to do something else, maybe `return true`.